incubator/hnc/e2e/tests/delete_crd_test.go

```diff
@@ -0,0 +1,43 @@
+package test
+
+import (
+	"os/exec"
+	"os"
+	"time"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Delete-anchor-crd", func() {
+	
+	AfterEach(func() {
+		// clean up
+		nses := []string{
+			"delete-crd-parent",
+			"delete-crd-child",
+		}
+
+		for _, ns := range nses {
+			runSuccess("kubectl hns set", ns, "-a")
+			//remove subname annotation
+			runSuccess("kubectl annotate ns", ns, "hnc.x-k8s.io/subnamespaceOf-")
+			runSuccess("kubectl delete ns", ns)
+		}
+
+		runSuccess("kubectl apply -f", os.Getenv("HNC_REPAIR"))
+		GinkgoT().Log(time.Now())
+	})
+
+	It("should create parent and deletable child, and delete the CRD", func() {
+		GinkgoT().Log(time.Now())
+		// set up
```

Comment:
Don't forget to `Skip()` if `HNC_REPAIR` isn't set.

Also, since the `"HNC_REPAIR"` string would now be used twice (if you follow this advice), it's a good idea to replace it with a constant so if you mistype it somewhere, the compiler will complain. Maybe at the top of this file (around line 12):

```
const (
  hncYamlEnvVar = "HNC_REPAIR"
)
```

Reply:
done